src/DependencyInjection/StoryblokExtension.php

```diff
@@ -55,6 +60,11 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('storyblok_api.webhooks.secret', $config['webhook_secret']);
         $container->setParameter('storyblok_api.version', $config['version']);
 
+        $container->setParameter('storyblok.controller.cache.public', $config['controller']['cache']['public']);
+        $container->setParameter('storyblok.controller.cache.must_revalidate', $config['controller']['cache']['must_revalidate']);
+        $container->setParameter('storyblok.controller.cache.max_age', $config['controller']['cache']['max_age']);
+        $container->setParameter('storyblok.controller.cache.smax_age', $config['controller']['cache']['smax_age']);
```

Comment:
Do we need all this as parameters ? they could be injected directly from the config no ?

Reply:
we could but why would you say they hurt? I tried to configure the services in the `services.php`